src/Handlers/PaymentResponseHandler.php

```diff
@@ -292,14 +299,23 @@ public function handleAdyenApis(
         PaymentResponseHandlerResult $paymentResponseHandlerResult
     ): array {
         $resultCode = $paymentResponseHandlerResult->getResultCode();
+        $refusalReason = $paymentResponseHandlerResult->getRefusalReason();
+        $refusalReasonCode= $paymentResponseHandlerResult->getRefusalReasonCode();
 
         switch ($resultCode) {
             case self::AUTHORISED:
+                [
+                    "isFinal" => true,
+                    "resultCode" => $this->paymentResponseHandlerResult->getResultCode()
+                ];
+                break;
```

Comment:
Is think this isn't necessary? Both vars are returned for REFUSED and ERROR as well